tests/storage-buckets

```diff
@@ -4,6 +4,13 @@ set -eux
 # Install LXD
 install_lxd
 
+if ! hasNeededAPIExtension storage_buckets; then
```

Comment:
Could we add this skip logic to the main script that runs each test? 
This way if you manually run the test against an incompatible host you dont get a zero exit status.

Reply:
Moving it to outside of the test script itself would make it impossible to do the API detection, no? In the main script, we only know about the snap channel, not the API extensions that come with it.

Reply:
In the long run, when I'll have `bin/lxd-run` ready, I intend to include a testrunner image cache that will allow dumping the api extentions from the testrunner prior to "forking" it to run the tests. This will permit API detection ahead of test runs but are not there yet :/